distribution/environments/aws-ec2-cloud/CloudFormation/cloudformation-template.json

```diff
@@ -0,0 +1,165 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Parameters" : {
+    "InstanceTypeParameter" : {
+      "Type" : "String",
+      "Default" : "t2.large",
+      "AllowedValues" : ["t2.micro", "m1.small", "m1.large", "t2.large"],
```

Comment:
I should change that. Allowing a `t2.micro` probably makes no sense. But then, should I actually list *all* allowed values, not sure.

Reply:
I tend to agree, though we also know that running a master on 512MB of RAM would very likely be a recipe for failure. I couldn't find the predefined type for instance types, though. Care to point me to it? I agree it looks weird it's not defined, but I cannot find it in https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#aws-specific-parameter-types @mandie722

Reply:
It's under the Elastic section: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Instance.html

Reply:
@mandie722 seems more something usable when calling remote API, here XML AIUI when going on the higher levels of that subdocumentation? What type/value would I use here in e cloudformation template?